src/edu/stanford/nlp/sempre/interactive/ActionLanguageAnalyzer.java

```diff
@@ -0,0 +1,123 @@
+package edu.stanford.nlp.sempre.interactive;
+
+import java.util.Arrays;
+
+import edu.stanford.nlp.sempre.LanguageAnalyzer;
+import edu.stanford.nlp.sempre.LanguageInfo;
+
+/**
+ * supporting the interactive language
```

Comment:
to tokenize [], {}, =, >=, and other core language specific/domain specific things.